app/src/main/java/com/futurice/freesound/feature/search/SoundItemViewHolder.java

```diff
@@ -121,4 +129,46 @@ protected DataBinder getViewDataBinder() {
         return viewDataBinder;
     }
 
+    static class SoundItemViewHolderFactory extends ViewHolderFactory {
+
+        @NonNull
+        private final Picasso picasso;
+
+        @NonNull
+        private final SchedulerProvider schedulerProvider;
+
+        SoundItemViewHolderFactory(@NonNull final Context context,
+                                   @NonNull final Picasso picasso,
+                                   @NonNull final SchedulerProvider schedulerProvider) {
+            super(context);
+            this.picasso = picasso;
+            this.schedulerProvider = schedulerProvider;
+        }
+
+        @NonNull
+        @Override
+        public RecyclerView.ViewHolder createViewHolder(@NonNull final ViewGroup parent) {
+            View view = LayoutInflater.from(parent.getContext())
+                                      .inflate(R.layout.view_sound_item, parent, false);
+            return new SoundItemViewHolder(view, picasso, schedulerProvider);
+        }
+    }
+
+    static class SoundItemViewHolderBinder implements ViewHolderBinder {
+
+        @NonNull
+        private final SoundItemViewModelFactory viewModelFactory;
+
+        SoundItemViewHolderBinder(@NonNull final SoundItemViewModelFactory viewModelFactory) {
+            this.viewModelFactory = viewModelFactory;
+        }
+
+        @Override
+        public void bind(@NonNull final RecyclerView.ViewHolder viewHolder,
+                         @NonNull final DisplayableItem item) {
+            SoundItemViewHolder soundItemViewHolder = SoundItemViewHolder.class.cast(viewHolder);
```

Comment:
Not that I know of, just a different way for casting objects that I personally find more natural/readable :)